test/e2e/chainsaw/operator/single-tenancy/test-scenarios/invalid-podtemplatespec/assert-mcpserver-invalid-condition.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: toolhive.stacklok.dev/v1alpha1
```

Comment:
I think we can probably avoid doing the heavyweight e2e tests and instead do them in integration tests. As we only really need to add an `MCPServer` and make assertions. They're pretty much built for these type of scenarios.